javascript/javascript1/week3/main.js

```diff
@@ -0,0 +1,83 @@
+// Item array removal
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+names.splice(1, 1);
```

Comment:
Splicing based on the index is not the right approach here. You need to loop through the array, search for the name and then remove it.